README.md

```diff
@@ -1,3 +1,56 @@
-# Locale Switcher
+<p align="center">
+  <img src="./icons/128.png" alt="logo" />
+</p>
 
-Chrome Extension to quickly switch your browser locale.
+# 🌏 Locale Switcher
+
+Browser Extension to quickly change your browser locale.
+
+## ❓ What does it do?
+
+When a locale chosen the following thinks will be adjusted:
+
+- `navigator.language`
+- `navigator.languages`
+- `Accept-Language` headers
+
+## ✨ Features
+
+- Chrome & Firefox
+- Support for 700+ languages and variations
+- Per tab basis
```

Comment:
Perhaps: Changes the locale for the current tab only

Reply:
But you can change multiple tabs and it persists also, so tecnically it's scoped on the tab and not the current. Then again it might be easier to understadnt